### PR TITLE
Dev/error

### PR DESCRIPTION
**Pull request by AvinashMahanthi**
### Functionality:
Throw error whenever there is an internal or external error.

### Solution:
Add Error Component, Error Handler Service and error xml

### Risk level:
- [X] high 
- [ ] medium
- [ ] low

### How to test:

- ng serve
- firebase emulators: start
- Press Submit button without filling input fields.
- Then you can see the Error.

### ScreenShots:

image

image

### CODE REVIEW

**Comment by viv9k** [comment]
If there is no sub component then 0000 should be shown. There should be 12 digits of error code.

**Comment by viv9k** [comment]
If there is no sub component then 0000 should be shown. There should be 12 digits of error code.

File: src/app/app.module.ts [hunk 1]
```diff
@@ -77,7 +80,8 @@ import { RangeCardComponent } from "./body/my-dashboard/performance-chart/range-
     AngularFireFunctionsModule,
     AngularFirestoreModule,
     AppRoutingModule,
-    GoogleChartsModule.forRoot({ version: 'current' })
+    GoogleChartsModule.forRoot({ version: 'current' }),
+    HttpClientModule,
```
Comment (bighnesh0404) [inline src/app/app.module.ts:84]: remove the comma from here

**Review by bighnesh0404 — CHANGES_REQUESTED** [review]

**Comment by viv9k** [comment]
Please resolve merge conflict asap

**Comment by viv9k** [comment]
Please review this code asap
tomorrow we need to give release

**Review by viv9k — APPROVED** [review]
ship it

**Review by bighnesh0404 — APPROVED** [review]

**Pull request merged**